test/test_unistd.rs

```diff
@@ -120,6 +123,31 @@ macro_rules! execve_test_factory(
 );
 
 #[test]
+fn test_getcwd() {
+  // workaround for the fact that on os x TmpDir::new returns /var/folders/... but upon
+  // chdir into that directory getcwd returns /private/var/folders/...
+  let base = if cfg!(target_os = "macos") {
+    "/private/tmp/"
+  } else {
+    "/tmp/"
+  };
```

Comment:
This seems a little fragile.  I think it would be OK to just verify that the nix `getcwd` results match those of libstd without hardcoding absolute base paths which may change based on the OS and system (My tmpfs may not be at `/tmp`).